heartbeat/monitors/active/dialchain/net.go

```diff
@@ -64,6 +65,16 @@ func netDialer(timeout time.Duration) NetDialer {
 				return nil, err
 			}
 
+			portNum, err := strconv.Atoi(port)
```

Comment:
I wonder if this should be more a config validation already?

Reply:
unfortunately stdlib net uses string for ports in quite a lot of places. Kind of forces me to switch between string and uint16 every now and then :(

Reply:
Sorry, comment was on the wrong line. I was wondering about the valid port range.